### DISCUSSION TITLE
Video preview on discord without downloading

### DISCUSSION THREAD

**Discussion by JohnTa15** [JunkFood02/Seal#858] [Q&A]
Hello I have a question,
How can I download a video and share it to discord with preview feature(Without downloading it only)? Some times this is working. I also see that the mp4 file is supported to preview but in these case my clips I download from seal does not.

**Comment by Marco-9456** [comment]
Check the format for the videos that are always working and when you download a video in seal next time enable format selection and download the video with that format if available, seal is downloading video in original format without converting it so this is an external issue.

**Reply by JohnTa15** [reply]
Most vids that I download are from TikTok so I think by default they are mp4 or webm
